### PR TITLE
Lb/enable RSpec/VerifiedDoubles

### PR DESCRIPTION
**Pull request by elceebee**
## Context
We ant to enable all the rspec rules for rubocop.

## Changes proposed in this pull request
This PR removes unverified doubles

## Guidance to review
Have a loop
The tests should pass!

## Things to check

- [ ] If the code removes any existing feature flags, a data migration has also been added to delete the entry from the database
- [ ] This code does not rely on migrations in the same Pull Request
- [ ] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [ ] If this code adds a column to the DB, decide whether it needs to be in analytics yml file or analytics blocklist
- [ ] API release notes have been updated if necessary
- [ ] If it adds a significant user-facing change, is it documented in the [CHANGELOG](CHANGELOG.md)?
- [ ] Required environment variables have been updated or [added to the Azure KeyVault](/docs/environment-variables.md#deploy-pipeline)

### CODE REVIEW

File: spec/forms/user_invite_form_spec.rb [hunk 1]
```diff
@@ -71,10 +71,9 @@
         }
       end
 
-      before "it sends invitation" do
-        user_mailer = double(:user_mailer)
-        expect(UserMailer).to receive(:invitation_email) { user_mailer }
-        expect(user_mailer).to receive(:deliver_later)
+      it "sends invitation" do
+        user_invite_form = described_class.new(form_params)
+        expect { user_invite_form.invite }.to have_enqueued_mail(UserMailer, :invitation_email)
```
Comment (Nitemaeric) [inline spec/forms/user_invite_form_spec.rb:76]: 🙌

**Review by Nitemaeric — APPROVED** [review]
Nice one!

**Review by CatalinVoineag — APPROVED** [review]

**Review by JamieCleare2525 — APPROVED** [review]
All looks good.
Might be a moving target though since https://github.com/DFE-Digital/itt-mentor-services/pull/222 was merged

**Comment by elceebee** [comment]
> All looks good. Might be a moving target though since #222 was merged

Already took care of it!

**Pull request merged**